kustomize/skipper/prometheus-skipper/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: flagger-system
+  annotations:
+    linkerd.io/inject: disabled
+  labels:
+    istio-injection: disabled
```

Comment:
Those labels sound unrelated to skipper?

Reply:
thanks - removed